src/logic/safe/transactions/gas.ts

```diff
@@ -62,6 +62,7 @@ type TransactionEstimationProps = {
   approvalAndExecution?: boolean
 }
 
+// TODO: Delete as it is not used anymore
```

Comment:
Is this to be tackled in this PR? Please modify it to `@TODO:` to match JSDOC formatting.

Reply:
I think we can remove it in this PR and also remove `estimateGasForTransactionApproval` since we're only estimating for transaction executions now.